yaml/xyz/openbmc_project/State/BMC/Redundancy/Sibling.interface.yaml

```diff
@@ -0,0 +1,95 @@
+description: >
+    This interface describes the API to read information from a sibling BMC in a
+    redundant configuration.
+
+properties:
+    - name: BMCPosition
+      type: size
+      default: 0
+      flags:
+          - readonly
+      description: >
+          The position the sibling BMC thinks it is.  Important on systems where
+          there is an opportunity to plug hardware or cables in wrong.
+
+    - name: FWVersion
+      type: string
+      flags:
+          - readonly
+      description: >
+          The version of the firmware that the sibling is running.
+
+    - name: Provisioned
+      type: boolean
+      default: false
+      flags:
+          - readonly
+      description: >
+          If the sibling has been provisioned.
+
+    - name: RedundancyEnabled
+      type: boolean
+      default: false
+      flags:
+          - readonly
+      description: >
+          If the sibling BMC has redundancy enabled.  This may only make sense
+          if the sibling has the active role.
+
+    - name: FailoversPaused
```

Comment:
Posted generic comment in https://gerrit.openbmc.org/c/openbmc/phosphor-dbus-interfaces/+/73944/5/yaml/xyz/openbmc_project/State/BMC/Redundancy/Sibling.interface.yaml#47

Just, sharing MoM from the workshop...

https://ibm.ent.box.com/notes/1467347410923

```
Q: Initiate Failover and Pause sync and reenable it once P-BMC comes back during the failover reset..

Dean/Ramesh: Lets keep it like reboot of the P-BMC, pause and initiate sync onces comes back

Amit: If the P-BMC did not comes back??

Dean: Disable Redundancy bit
```

Reply:
Clarified in the upstream review.